### DISCUSSION TITLE
Workshop outcomes

### DISCUSSION THREAD

**Discussion by ml-evs** [MADICES/MADICES-2022#21] [Ideas]
I think we should come up with concrete bulletpoints to summarise the workshop outcomes to put on the website (I think we had this in the google doc).

Here is my suggestion:

1) To prepare a perspective paper targeted at providing guidelines and recommendations for new projects in our field that work with data.
    i) For example, how best to prepare and disseminate datasets, databases and APIs such that they are interoperable with existing projects.
    ii) Focusing on the area of X-ray absorption spectroscopy as a concrete example of a domain where a sample, its context/metadata and relations with other samples and computational experiments (e.g. peak assignment) are necessary for the science
2) To identify particular challenges that can be overcome via new collaborative software libraries/infrastructure and motivate their creation over follow-up meetings/hackathons
    i) e.g., services that aid discoverability of apps/schemas, and ways of discovering if two apps are interoperable
    ii) e.g., Ways to progressively adopt open linked data principles that are accessible to non-experts

**Comment by kjappelbaum** [comment]
Sounds good, do we want to add something about "3. Identify and foster potential collaborations"

**Reply by ml-evs** [reply]
Tried to word 2) in that way, but a separate point is also fine with me. I know @stefhk3 had thoughts about targeting real solutions in this workshop, do you have anything you would add Stefan?

**Reply by stefhk3** [reply]
The more hands-on it gets, the better. That's not easy, but we should try. I didn't see that item, sorry. I think the two points are good.  The collaborations could go along the lines of initiatives, say NFDI4Chem working with ? If we can say we have set up x networks, that would be really good.

**Reply by ml-evs** [reply]
Perhaps something like

3) To motivate follow-up meetings, hackathons and foster new cross-initiative collaborations.

Part of the "problem" is that there are already several big collaborations (all the NFDIs) that have critical mass to go their own way.

> The more hands-on it gets, the better.

I agree - I think the final day breakout rooms should basically be either drafting the paper, or smaller hackathon style stuff, e.g. trying to ingest each other's data, hacking out ideas for infra, all under the "neutral" MADICES banner. Of course we will have to see how it goes, there may be more discussion to have. I may suggest a breakout room solely around collaborative software developments.

**Reply by kjappelbaum** [reply]
>Part of the "problem" is that there are already several big collaborations (all the NFDIs) that have critical mass to go their own way.

That is clear, however it might be nice if there would be also some synergies between the different existing collaborations (to avoid too much duplication). And, at least for some of the developments, interoperability will still end up being a problem :D 

> I may suggest a breakout room solely around collaborative software developments.
Perhaps it would be best if we could already come up with a problem. I might even I imagine that someone of us shows a quick tutorial  using some of our favorite tools (maybe a bit along the lines of Donny's course) and then we work together on some dataset (i.e., we would already write "bring your dataset" in the pre-workshop email)

**Comment by kjappelbaum** [comment]
Let's add on the main page below the description?

**Reply by ml-evs** [reply]
Maybe above?

**Reply by kjappelbaum** [reply]
also ok :) let's just put it somewhere ;)

**Reply by kjappelbaum** [reply]
https://github.com/MADICES/MADICES.github.io/pull/11

**Comment by kjappelbaum** [comment]
maybe we can also think about a list of more concrete tools we currently see as missing and which we/MADICES could collaboratively develop.
